uitools/common/src/OverviewMapController.cpp

```diff
@@ -111,6 +111,42 @@ namespace Esri::ArcGISRuntime::Toolkit {
 
     if (auto sceneView = qobject_cast<SceneViewToolkit*>(m_geoView))
     {
+      // create a lambda to handle setting up the inset map and viewpoint
+      auto setupInsetMapForScene = [this](SceneViewToolkit* sceneView)
+      {
+        // create the map
+        auto map = new Map(BasemapStyle::ArcGISTopographic, m_insetView);
+        // set the initial viewpoint (scale = main scene's scale * scaleFactor)
+        const Viewpoint viewpoint = sceneView->currentViewpoint(ViewpointType::CenterAndScale);
+        const Viewpoint newViewpoint{
+                                     geometry_cast<Point>(viewpoint.targetGeometry()),
+                                     viewpoint.targetScale() * scaleFactor()};
+
+        // set the initial viewpoint before setting on the mapview
+        map->setInitialViewpoint(sceneView->arcGISScene()->initialViewpoint());
+        m_insetView->setMap(map);
+      };
+
+      // if the map is already loaded, setup the inset map
+      if (sceneView->arcGISScene()->loadStatus() == LoadStatus::Loaded)
+      {
+        setupInsetMapForScene(sceneView);
+      }
+      // set up the inset map once the scene is done loading
+      else
+      {
+        connect(sceneView->arcGISScene(), &Scene::doneLoading, this, [setupInsetMapForScene, sceneView](Error e)
```

Comment:
Double check this compiles, but you can avoid a hefty copy here.

```suggestion
        connect(sceneView->arcGISScene(), &Scene::doneLoading, this, [setupInsetMapForScene = std::move(setupInsetMapForScene), sceneView](Error e)
```